package.json

```diff
@@ -29,7 +29,8 @@
     "build:styles": "postcss \"src/components/**/*.css\" --dir lib/ --base src/ --verbose",
     "chromatic": "chromatic",
     "commit": "./node_modules/.bin/cz",
-    "copy-to-lib": "yarn copy-icons-to-lib && yarn copy-tokens-to-lib",
+    "copy-to-lib": "yarn copy-icons-to-lib && yarn copy-tokens-to-lib && yarn copy-fonts-to-lib",
+    "copy-fonts-to-lib": "copyfiles -u 3 src/design-tokens/tier-1-definitions/fonts.css src/design-tokens/tier-1-definitions/fonts/**/* lib/tokens",
```

Comment:
little bit of finagling so that traject can import from `@chanzuckerberg/eds/lib/tokens/font.css`. It's only imported in like 2 spots though, so we can drop this if we think it's too gross